fe/.eslintrc.yml

```diff
@@ -0,0 +1,56 @@
+env:
+  browser: true
+  es2021: true
+extends:
+  - airbnb-typescript
+  - airbnb/hooks
+  - plugin:@typescript-eslint/recommended
+parser: "@typescript-eslint/parser"
+parserOptions:
+  ecmaFeatures:
+    jsx: true
+  ecmaVersion: 12
+  sourceType: module
+  files:
+    - "*.ts"
+    - "*.tsx"
+  project: tsconfig.json
+plugins:
+  - react
+  - "@typescript-eslint"
+rules:
+  comma-dangle:
+    - error
+    - only-multiline
+  import/no-cycle: off
+  "@typescript-eslint/explicit-function-return-type": "off"
+  "@typescript-eslint/no-unused-vars": warn
+  linebreak-style: "off"
+  import/no-unresolved: "off"
+  import/extensions: "off"
+  react/jsx-filename-extension:
+    - "off"
+    - extensions:
+        - ".js"
+        - ".jsx"
+  react/destructuring-assignment: "off"
+  jsx-a11y/no-static-element-interactions: "off"
+  prefer-template: "off"
+  no-plusplus: "off"
+  eol-last: "off"
+  react/prop-types: "off"
+  arrow-parens:
+    - Error
+    - as-needed
+  max-len:
+    - error
+    - code: 200
```

Comment:
Any strong opinions on this? Usually this is a controversial topic. Some people like their lines short, some long.

Reply:
i have usually gone with 120 but thats a intellij default and i have larger font size for teaching usually. 200 would be absolute max i guess. you can fit 146 in vsc with normal size and sidebar and 175 without sidebar so perhaps on the lines of that. I personally think one should not need to scroll sideways.

Reply:
Tried in my editor and 200 does seem a bit extreme, changed to 135